x/staking/keeper/params.go

```diff
@@ -10,35 +10,35 @@ import (
 )
 
 // UnbondingTime
-func (k Keeper) UnbondingTime(ctx sdk.Context) (res time.Duration) {
-	k.paramstore.Get(ctx, types.KeyUnbondingTime, &res)
-	return
+func (k Keeper) UnbondingTime(ctx sdk.Context) time.Duration {
+	params := k.GetParams(ctx)
+	return params.UnbondingTime
```

Comment:
nit: no need to store into variable, just do `return k.GetParams(ctx.<FieldName>)`